README.md

```diff
@@ -319,8 +319,7 @@ class LitAutoEncoder(pl.LightningModule):
         super().__init__()
         self.automatic_optimization = False
 
-    def training_step(self, batch, batch_idx, optimizer_idx):
-        # access your optimizers with use_pl_optimizer=False. Default is True
```

Comment:
the comment can stay yes?

Reply:
comment says to use `use_pl_optimizer=False` but actually it uses `use_pl_optimizer=True` down below.